README.rst

```diff
@@ -24,6 +24,26 @@ Code Overview
 
 The program ``pipelines.py`` creates a graph using the Python package ``networkx``, and then uses the Ocean software tools to run the ``minimum_vertex_cover`` function from within the ``dwave_networkx`` package.
 
+Inside the Code: The Lagrange Parameter
+------ --- ----  --- -------- ---------
+Gamma, our Lagrange parameter, weights the constraints in the problem versus
+the energy term (objective). If the Lagrange parameter is too small, relative
+to the strength of the energy term, the constraints may be violated.
+
+A valid vertex cover is a subset of nodes in the graph such that every edge
```

Comment:
Do we really need this paragraph? The definition of vertex cover seems a bit late here, and the rest raises the question of what happens in relevant problems that one can't eyeball a solution. Why not simply state that the given example needs 4 nodes but with the default Lagrange the code frequently gave invalid solutions with fewer nodes?

Reply:
In my opinion, yes, we do. It makes it clearer, in the discussion coming up. Every time I look at vertex cover, I have to re-familiarize myself with the math problem and the definition of it.

Reply:
Personally I'm not persuaded but I'll defer to @m3ller. However, if it is kept, its current location breaks the discussion of Lagrange into two paragraphs. Would it not make more sense to put it before the two Lagrange paragraphs so that explanation is continuous?

Reply:
@JoelPasvolsky and @m3ller, I reorganized the section so that the discussion of the size of the min vertex cover is first, followed by the discussion of Gamma. Perhaps that takes care of it.